__tests__/app.test.js

```diff
@@ -120,6 +122,16 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe("PATH NOT FOUND");
       });
   });
+  test("GET:200 Responds with articles filtered by topic", () => {
+    return request(app)
+      .get(`/api/articles?topic=mitch`)
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
```

Comment:
how about error test for topic that doesn't exist?   And also test to expect status 200 and empty array returned when topic exists in the db but has no articles yet? 
This behaviour will require you to check a topic exists (perhaps use the fetchTopics endpoint you made at the start, or a checkIfExists function if you've made one already.)